.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -118,10 +123,15 @@ steps:
           - "heartbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "Heartbeat: Go Integration Tests"
+              context: "heartbeat: Go Integration Tests"
 
-      - label: ":ubuntu: Heartbeat Python Integration Tests"
+      - label: ":ubuntu: Heartbeat Python (Module) Integration Tests"
         command: |
+          set -euo pipefail
+          # defines the MODULE env var based on what's changed in a PR
+          source .buildkite/scripts/changesets.sh
+          defineModuleFromTheChangeSet heartbeat
```

Comment:
FYI there is no module directory under heartbeat, as as a result we are getting `find: ‘heartbeat/module’: No such file or directory`: https://buildkite.com/elastic/heartbeat/builds/4736#018f588a-18bb-490e-96e2-6bec92e1d2ce/140-149

Reply:
The behaviour was same on Jenkins. But as discussed, changeset.sh was reworked and this issue fixed in https://github.com/elastic/beats/pull/39611